codes/classical/q-ary_digits/rank_modulation.yml

```diff
@@ -0,0 +1,43 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+
+# code id, physical, logical are all lower case
+code_id: rank_modulation
+physical: groups
+logical: q-ary_digits
+
+
+name: 'Rank modulation code'
+introduced: '\cite{doi:10.1109/ISIT.2008.4595285}'
+
+
+description: |
+    Rank modulation codes are a family of codes which encode an alphabet of size \(q\) into a set \(S_n\) of permutations of \([n]=(1,2,...,n)\).
+    They can be derived from Lee-metric codes, Reed-Solomon codes, quadratic residue codes and most binary codes.
+
+protection: |
+    Protects against \textit{Kendall} errors, which are operators \(e\) on the Kendall metric space \(\mathcal(X_n)=(S_n, d_\tau)\) satisfying \(d_\tau(\sigma, e(\sigma))=1,\forall\sigma\in S_n\). 
+    The Kendall tau distance \(d_\tau(\sigma, \pi\) is defined by the minimal number of swaps of adjacent elements so as to change \(\sigma\) into \(\pi\).
+
+
+features:
+
+  rate: some rank modulation codes with code distances \(d=\Theta(n^{1+\epsilon}), \epsilon\in[0,1]\) can achieve a rate of \(1-\epsilon\) (\cite{doi:10.1109/ISIT.2010.5513604})
+
+
+realizations:
+  - Flash memory
```

Comment:
refs needed

Reply:
A description of the application, together with some references, is given in 10.1109/ISIT.2010.5513604